apps/ocpp/management/commands/export_transactions.py

```
@@ -4,56 +4,54 @@
+class Command(BaseCommand):
+    help = "Export OCPP transactions and related data to a JSON file"
 
-        with open(options["output"], "w", encoding="utf-8") as fh:
-            json.dump(data, fh, indent=2, ensure_ascii=False)
+    def add_arguments(self, parser) -> None:
+        add_transactions_export_arguments(parser)
 
-        self.stdout.write(
-            self.style.SUCCESS(f"Exported {len(data['transactions'])} transactions")
-        )
+    def handle(self, *args, **options):
```

Ruff (0.15.1)

[warning] Unused method argument: `args`

(ARG002)